src/components/FileTree/index.tsx

```diff
@@ -78,7 +78,9 @@ export class FileTreeBase extends React.Component<Props> {
     const { onSelect, version } = this.props;
 
     if (version) {
-      return <FileTreeNode {...props} onSelect={onSelect} version={version} />;
+      return (
+        <FileTreeNode {...props} onSelect={onSelect} versionId={version.id} />
```

Comment:
This and the change in `FileTreeNode` go against https://github.com/mozilla/addons-code-manager/pull/400#discussion_r267058168

Reply:
Yes, it does go against it which is why I filed https://github.com/mozilla/addons-code-manager/issues/754 to fix it. There's a todo comment about it `mapStateToProps()`.

The problem here is that we can't rely on a passed `version` object *and* be guaranteed a re-render when `version.focusedPath` changes. We need to map the object to state to guarantee an update.